client/src/views/ProtubeScreen.vue

```diff
@@ -135,4 +204,39 @@ socket.on("new-video-timestamp", async (newStamp) => {
     if ((await player.getPlayerState()) === 2) player.playVideo();
   }
 });
+
+// retrieving the queue and stop skeletonloading
+socket.on("connect", async () => {
+  const data = await new Promise((resolve) => {
```

Comment:
You emit the queue-update on a connection in ScreenSocket.js (server) so you don’t need to fetch it here as well.

Reply:
Yes, fixed now :)